src/Dockerfile

```diff
@@ -34,7 +34,9 @@ RUN apk add --no-cache \
     tini \
     tzdata \
     unzip \
-    wget
+    wget \
+    # Install grep to avoid issues in pihole -w/b with the default busybox grep
+    grep
```

Comment:
The other packages are in alphabetical order

Reply:
Fussy!